### ISSUE TITLE
Proposal: Create aliases for default OTP versions

### ISSUE THREAD

**Issue by Gigitsu** [erlef/docker-elixir#71]
In my CI script, it would be handy to specify not only the elixir version but also the OTP version explicitly. 

So, for example, for the latest elixir release, it would be handy to have `1.14.4-otp-25` alongside the already existing `1.14.4, 1.14, latest` (the same for the variants `slim` and `alpine`) even if the OTP 25 is the default one for the elixir 1.14.

As far as I can see looking at the code, this will only require adding some aliases in [generate-stackbrew-library.sh](https://github.com/erlef/docker-elixir/blob/master/generate-stackbrew-library.sh). I would be happy to make a PR for this, if my guess of the aliases is right.

**Comment by idyll** [comment]
I think this has been done now,  when I look at docker hub I see OTP specific versions.
I think it's important to continue to provide these labels even after the launch of OTP26 so I'm going to add comments below, but my suspicion is that this issue should just be closed.

**I think labelling the OTP version is critically important.**

I'd go further though, I think what we really need is:
 - `1.14-otp-25`
 - `otp-25` (aka latest-otp-25)

I know several people who've run into problems in the last week because the 1.14 changed from OTP25 to OTP26. 

Often I will use the OTP image as a base image for runtime as it has fewer dependencies than the Elixir image:

```#dockerfile
FROM elixir:1.14-alpine as build
...
FROM erlang:25-alpine
...
```

As you can imagine this causes problems when OTP changes. 
A move to including OTP version in the tag would eliminate this issue:

```#dockerfile
FROM elixir:1.14-otp-25-alpine as build
...
FROM erlang:25-alpine
...
```

**Comment by nathany-copia** [comment]
For reproducible builds, I agree that using the explicit Elixir and OTP versions should be encouraged.

**Comment by Gigitsu** [comment]
@idyll now I see the tag `1.14.5-otp-25` but just because the new default is `otp-26` and the previous patch version (`1.14.4`) is gone.

I totally agree with you and @nathany-copia that explicit Elixir and OTP versions should be used even for latest versions

**Comment by Gigitsu** [comment]
I've created the PR #82 to address this issue

**Comment by nathany-copia** [comment]
Today we found that this issue extends beyond Elixir/Erlang. 

Upgrading from OTP 25 to OTP 26 also upgraded Alpine from 3.7.x to 3.8.x -- which isn't entirely obvious that it's happening. We ran into some connection issues with Alpine 3.18.2, but initially thought it was an issue with Erlang OTP 26.

[hexpm/elixir](https://hub.docker.com/r/hexpm/elixir) provides the control we want, allowing us to upgrade Alpine, Erlang, and Elixir independently.

It would be nice if the official images did the same.
